### PR TITLE
ISPN-11696 creating caches via operator

### PR DESCRIPTION
**Pull request by oraNod**
https://issues.redhat.com/browse/ISPN-11696

### CODE REVIEW

File: documentation/asciidoc/topics/proc_creating_caches_templates.adoc [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+spec:
+  adminAuth:
+    secretName: basic-auth <2>
+  clusterName: {example_crd_name} <3>
+  name: mycache <4>
+  templateName: org.infinispan.DIST_SYNC <5>
+----
++
+<1> Names the **Cache** CR.
+<2> Specifies the authentication secret with credentials for accessing {brandname} clusters.
```
Comment (rigazilla) [inline documentation/asciidoc/topics/proc_creating_caches_templates.adoc:39]: maybe we can specify the default value for keys in the secret? I mean `username` and `password`

Comment (oraNod) [reply documentation/asciidoc/topics/proc_creating_caches_templates.adoc:39]: @rigazilla I'm not sure I follow. should we rephrase as "Specifies the username and password to access {brandname} clusters." ?

Comment (rigazilla) [reply documentation/asciidoc/topics/proc_creating_caches_templates.adoc:39]: @oraNod I mean if we need to describe the format of the credential secret: i.e. by default the operator is expecting to found the username in the secret entry with key `username` and the password in the secret entry with key `password`. User can override this default specifying a complete `adminAuth` field, like this:
```
    username:
      key: username     <- keyname for username in secret
      name: basic-auth <- secretname
    password:
      key: password    <- keyname for password in secret
      name: basic-auth <- secretname for password
```

**Comment by rigazilla** [comment]
merged, thanks @oraNod !

**Pull request merged**